### ISSUE TITLE
Balanced::Customer uninitialised constant error ruby 

### ISSUE THREAD

**Issue by sithara** [balanced/balanced-ruby#159]
now all of a sudden I'm getting an error when I try to access Balanced::Customer 

Its giving me unintialized constant.

**Comment by mjallday** [comment]
This is not enough information to debug this issue. 
- which version of balanced-ruby are you using?
- can you provide a gist reproducing this error consistently using a clean environment?

**Comment by sithara** [comment]
balanced (0.7.4)
ruby 2.0.0
rails 4.0.2

rails console
Loading development environment (Rails 4.0.2)
2.0.0p247 :001 > require "balanced"
 => false 
2.0.0p247 :002 >  Balanced::Credit
 => Balanced::Credit 
2.0.0p247 :003 >  Balanced::Debit
 => Balanced::Debit 
2.0.0p247 :004 >  Balanced::Customer
NameError: uninitialized constant Balanced::Customer
    from (irb):4
    from /home/sithara/.rvm/gems/ruby-2.0.0-p247@watoozi/gems/railties-4.0.2/lib/rails/commands/console.rb:90:in `start'
    from /home/sithara/.rvm/gems/ruby-2.0.0-p247@watoozi/gems/railties-4.0.2/lib/rails/commands/console.rb:9:in`start'
    from /home/sithara/.rvm/gems/ruby-2.0.0-p247@watoozi/gems/railties-4.0.2/lib/rails/commands.rb:62:in `<top (required)>'
    from bin/rails:4:in`require'
    from bin/rails:4:in `<main>'
2.0.0p247 :005 >

**Comment by sithara** [comment]
balanced (0.7.4)
ruby 2.0.0
rails 4.0.2

```
rails console
Loading development environment (Rails 4.0.2)
2.0.0p247 :001 > require "balanced"
 => false 
2.0.0p247 :002 >  Balanced::Credit
 => Balanced::Credit 
2.0.0p247 :003 >  Balanced::Debit
 => Balanced::Debit 
2.0.0p247 :004 >  Balanced::Customer
NameError: uninitialized constant Balanced::Customer
    from (irb):4
    from /home/sithara/.rvm/gems/ruby-2.0.0-p247@watoozi/gems/railties-4.0.2/lib/rails/commands/console.rb:90:in `start'
    from /home/sithara/.rvm/gems/ruby-2.0.0-p247@watoozi/gems/railties-4.0.2/lib/rails/commands/console.rb:9:in `start'
    from /home/sithara/.rvm/gems/ruby-2.0.0-p247@watoozi/gems/railties-4.0.2/lib/rails/commands.rb:62:in `<top (required)>'
    from bin/rails:4:in `require'
    from bin/rails:4:in `<main>'
2.0.0p247 :005 > 
```

**Comment by mjallday** [comment]
Looks like the customer resource was implemented in 0.7.1 so it's not a version issue.

Can you run https://github.com/balanced/balanced-ruby/blob/0.7.4/examples/customers.rb  ?

I'd be very surprised if this is an issue with the gem itself since the [tests all pass](https://github.com/balanced/balanced-ruby/blob/0.7.4/spec/balanced/resources/customer_spec.rb). 

Have you tried starting with the basics and uninstalling and reinstalling the gem? Also, what do you get if you do `Balanced::VERSION`?

**Comment by sithara** [comment]
Sorry Marshall.I was looking at my gem list but my balanced version was
locked at 0.6.I upgraded it to 0.7 and got it working.
